### PR TITLE
Avatar: anonymous user

### PR DESCRIPTION
**Pull request by driesd**
### Description

This PR extends the `Avatar` component with a fallback for anonymous users. Also, a `creatable` prop was implemented to render a `user plus` icon.

#### Screenshots
Screenshot 2019-12-04 08 16 03

Screenshot 2019-12-04 08 23 59

### Breaking changes

`AvatarImage`: removed `imageClassName` prop.

### Follow up todo
Refactor duplicate code

### CODE REVIEW

File: src/components/avatar/AvatarInitials.js [hunk 1]
```diff
@@ -17,29 +32,33 @@ class AvatarInitials extends PureComponent {
...
     );
   }
 }
 
 AvatarInitials.propTypes = {
   /** Component that will be placed top right of the avatar image. */
   children: PropTypes.any,
-  /** The color of the avatar. */
-  color: PropTypes.oneOf(['neutral', 'mint', 'aqua', 'violet', 'teal', 'gold', 'ruby']),
+  /** The uui to determine the background color of the avatar. */
```
Comment (kevinwaelkens) [inline src/components/avatar/AvatarInitials.js]:
```suggestion
  /** The uuid to determine the background color of the avatar. */
```
✏️ ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,10 +4,14 @@
 
 ### Changed
 
+- [BREAKING] `Avatar`: added `creatable` prop with default value `false`. ([@driesd](https://github.com/driesd) in [#735](https://github.com/teamleadercrm/ui/pull/735))
```
Comment (ArnaudWeyts) [inline CHANGELOG.md]: Is this really a breaking change?

Comment (rathesDot) [reply CHANGELOG.md]: I would say the `creatable` prop itself is not a breaking change. But what about the fact that `Avatar` now has a anon user default, is that breaking or just an added feature?

Comment (driesd) [reply CHANGELOG.md]: The `creatable` prop addition is not breaking at all, it was a copy/paste mistake 🙈

Comment (rathesDot) [reply CHANGELOG.md]: Then it's time to remove it :-D

**Pull request merged**